rllib/algorithms/impala/torch/impala_torch_policy_rlm.py

```diff
@@ -0,0 +1,167 @@
+import logging
+from typing import Dict, List, Union
+
+from ray.rllib.algorithms.impala.torch.vtrace_torch_v2 import (
+    make_time_major,
+    vtrace_torch,
+)
+from ray.rllib.algorithms.ppo.ppo_torch_policy import validate_config
+from ray.rllib.algorithms.ppo.torch.ppo_torch_rl_module import PPOTorchRLModule
+from ray.rllib.policy.sample_batch import SampleBatch
+from ray.rllib.utils.torch_utils import convert_to_torch_tensor
+from ray.rllib.policy.torch_mixins import (
+    EntropyCoeffSchedule,
+    LearningRateSchedule,
+)
+from ray.rllib.policy.torch_policy_v2 import TorchPolicyV2
+from ray.rllib.utils.annotations import override, Deprecated
+from ray.rllib.utils.framework import try_import_torch
+from ray.rllib.utils.torch_utils import (
+    explained_variance,
+    global_norm,
+)
+from ray.rllib.utils.typing import TensorType
+
+torch, nn = try_import_torch()
+
+logger = logging.getLogger(__name__)
+
+
+class ImpalaTorchPolicyWithRLModule(
+    LearningRateSchedule,
+    EntropyCoeffSchedule,
+    TorchPolicyV2,
+):
+    def __init__(self, observation_space, action_space, config):
+        validate_config(config)
+        TorchPolicyV2.__init__(self, observation_space, action_space, config)
+        # Initialize MixIns.
+        LearningRateSchedule.__init__(self, config["lr"], config["lr_schedule"])
+        EntropyCoeffSchedule.__init__(
+            self, config["entropy_coeff"], config["entropy_coeff_schedule"]
+        )
+
+    @Deprecated(new="ImpalaTorchLearner.compute_loss_per_module()", error=False)
+    @override(TorchPolicyV2)
+    def loss(
+        self,
+        model: PPOTorchRLModule,
+        dist_class,
+        train_batch: SampleBatch,
+    ) -> Union[TensorType, List[TensorType]]:
+        train_batch[SampleBatch.ACTIONS]
```

Comment:
can you leave a comment on why you need to access these keys but not do anything with them?

Reply:
As discussed offline, I mirrored the TF implementation here but was now able to remove all of these without breaking the learner test.